SourceX/DiabloUI/errorart.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+namespace dvl {
+
+extern const char *btnData;
```

Comment:
`extern const char *const`

Reply:
MSVC refused to compile because it has a 64k limit on strings, I had to change it to a 2D array instead to get it to compile. Could you check if there are any magic keywords that should be added to it's definition :)